website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,278 @@
+   the response of the API call. 
+
+   If the API response is an unrecoverable error, such as an Insufficient Capacity, Karpenter will delete the NodeClaim, mark that instance type as temporarily unavailable, and create another NodeClaim if necessary.
+   Example log:
+    ```
+    {"level":"INFO","time":"2024-06-22T02:24:19.028Z","logger":"controller","message":"launched nodeclaim","commit":"490ef94","controller":"nodeclaim.lifecycle",
+       "controllerGroup":"karpenter.sh","controllerKind":"NodeClaim","NodeClaim":
+       {"name":"default-sfpsl"},"namespace":"","name":"default-sfpsl","reconcileID":
+       "9c9dbc80-3f0f-43ab-b01d-faac6c29e979","provider-id":
+       "aws:///us-west-2b/i-01234567adb205c7e","instance-type":"c3.2xlarge","zone":
+       "us-west-2b","capacity-type":"spot","allocatable":{"cpu":"7910m",
+       "ephemeral-storage":"17Gi","memory":"13215Mi","pods":"58"}}
+    ```
+5. Karpenter watches for the instance to register itself with the cluster as a node, and updates the node's 
+   labels, annotations and taints to match what was defined in the NodePool and NodeClaim. Once this step is 
+   completed, Karpenter will remove the `karpenter.sh/unregistered` taint from the Node.
+
+   If this fails to succeed within 15 minutes, Karpenter will remove the NodeClaim from the cluster and delete
+   the underlying instance, creating another NodeClaim if necessary.
+   Example log: 
+   ```
+   {"level":"INFO","time":"2024-06-22T02:26:19.028Z","logger":"controller","message":"registered nodeclaim","commit":"525136f-dirty","controller":"nodeclaim.lifecycle",
+    "controllerGroup":"karpenter.sh","controllerKind":"NodeClaim",
+    "NodeClaim":{"name":"default-xmbww"},"namespace":"","name":"default-xmbww","reconcileID":"1f1a350b-6b3a-4927-9632-074ee4a095da","provider-id":"aws:///us-west-2c/i-01234567adb205c7e","Node":{"name":"ip-xxx-xxx-xx-xxx.us-west-1.compute.internal"}}
+   ```
+6. Karpenter continues to watch the node, waiting until the node becomes ready, has all its startup taints removed, 
+   and has all requested resources registered on the node. 
+   Example log:
+    ```
+    {"level":"INFO","time":"2024-06-22T02:24:52.642Z","logger":"controller","message":
+        "initialized nodeclaim","commit":"490ef94","controller":
+        "nodeclaim.lifecycle","controllerGroup":"karpenter.sh","controllerKind":
+        "NodeClaim","NodeClaim":{"name":"default-sfpsl"},"namespace":
+        "","name":"default-sfpsl","reconcileID":
+        "7e7a671d-887f-428d-bd79-ddf603290f0a",
+        "provider-id":"aws:///us-west-2b/i-01234567adb205c7e",
+        "Node":{"name":"ip-xxx-xxx-xx-xxx.us-west-1.compute.internal"},
+        "allocatable":{"cpu":"7910m","ephemeral-storage":"18242267924",
+        "hugepages-2Mi":"0","memory":"14320468Ki","pods":"58"}}
+    ```
+
+As they are booting up, NodeClaims are included in the scheduling simulation for both launch and termination flows.
```

Comment:
Should we add some detail on what the implication of this is for the end user, or maybe just drop this detail? The important thing here is that Karpenter won't launch multiple instances for the same set of pods, but I don't think anyone would expect that it would.

Reply:
I can drop it. I don't think they need to know